### PR TITLE
TAJO-1871: '-DskipTests' flag does not work

### PR DESCRIPTION
**Pull request by jihoonson**

### CODE REVIEW

File: tajo-storage/tajo-storage-pgsql/pom.xml [hunk 1]
```diff
@@ -236,7 +236,6 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
```
Comment (jinossy) [inline tajo-storage/tajo-storage-pgsql/pom.xml]: It is my fault. but your patch can't run tests
please override configuration as follows:
<configuration combine.self="override">

File: tajo-storage/tajo-storage-pgsql/pom.xml [hunk 2]
```diff
@@ -224,10 +224,11 @@
   <profiles>
     <profile>
       <!-- testing-postgresql-server only supports x86-64 -->
-      <id>64bit-tests</id>
+      <id>64bit-intel-tests</id>
       <activation>
         <os>
-         <arch>x86_64</arch>
+          <name>x86_64</name>
```
Comment (hyunsik) [inline tajo-storage/tajo-storage-pgsql/pom.xml]: As far as I know, ``name`` is intended to describe operation system name like ``linux`` or ``mac os x``.

**Comment by jihoonson** [comment]
I've tested with my linux desktop and MBP.

**Comment by jinossy** [comment]
+1 LGTM

**Comment by jihoonson** [comment]
Thanks for your review.

**Pull request closed without merging**